tracing-log/src/log_tracer.rs

```diff
@@ -28,6 +28,7 @@
 //! [ignore]: struct.Builder.html#method.ignore_crate
 use crate::{format_trace, AsTrace};
 use log;
+pub use log::SetLoggerError;
```

Comment:
If the error is boxed, this type doesn't need to be made public

Reply:
the errors in `tracing-log` are not boxed (which was the case before, I think — I'm surprised that wasn't a problem in the past)